manageiq-operator/Dockerfile

```diff
@@ -1,5 +1,18 @@
 # Build the manager binary
-FROM docker.io/library/golang:1.23 as builder
+FROM registry.access.redhat.com/ubi9/ubi-minimal as builder
+
+COPY go.mod /go.mod
```

Comment:
Do we need  go.sum?

Reply:
No, I was just reading the `go` line from the file to figure out which version we should download.

Reply:
Good. Thanks.